common/lib/src/realm_types.dart

```diff
@@ -49,30 +51,30 @@ class Mapping<T> {
   List<T?> getListOfNullables(RealmAccessorMarker accessor, RealmObjectMarker object, String propertyName) => accessor.getList<T?>(object, propertyName);
 }
 
-const _intMapping = Mapping<int>();
-const _boolMapping = Mapping<bool>();
+const _intMapping = Mapping<int>(indexable: true);
+const _boolMapping = Mapping<bool>(indexable: true); // oddly enough :-)
```

Comment:
Let's remove the comment :)

Reply:
So strict 😄 .. but it is odd. Anyway, I will remove